### DISCUSSION TITLE
When do you plan to release vnext to main?

### DISCUSSION THREAD

**Discussion by oliwerhelsen** [aws-samples/aws-saas-factory-eks-reference-architecture#57] [Q&A] [answered]
Hi Core-team,

Loving what you're doing with this project and I am curious about Vnext. When do you plan to release vnext to main? If not in the near future, what are the pros and cons of choosing Main for my next project?

Best regards
Oliwer H.

**Comment by ranjith-aws** [comment, marked as answer]
Hi @oliwerhelsen - Appreciate the feedback. We are in the middle of testing and documentation for vNext and are looking to launch before end of September. I will make sure to let you know here once we get closer to launching.

Ranjith

**Reply by oliwerhelsen** [reply]
Thanks, @ranjith-aws. Looking forward to the release.
